sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -479,6 +486,16 @@ class depends_on {
 }
 ----
 
+`ALL_LEAVES_SHORTCUT` provides a shortcut for adding all leaves of a graph as dependencies.
+Therefore if this wildcard node is passed to `depends_on` in the explicit API,
+the created node will depend on all the current leaves in the graph.  
+[source,c++]
+----
+const sycl::ext::oneapi::experimental::node ALL_LEAVES_SHORTCUT =
+    sycl::detail::createSyclObjFromImpl<sycl::ext::oneapi::experimental::node>(
+        nullptr);
+----
+
```

Comment:
I think that trying to achieve this without modifying APIs is admirable, but this feels very hacky to me. If we don't want to modify any existing APIs then perhaps just a new property which will depend on all leaves makes sense? This doesn't change the existing signature of `add()` and is part of the purpose of having an extensible property list.